test/LendingPool.t.sol

```diff
@@ -2243,6 +2259,10 @@ contract LiquidationTest is LendingPoolTest {
         assertEq(pool.totalRealisedLiquidity(), 0);
         assertFalse(pool.isTranche(address(jrTranche)));
         assertFalse(pool.isTranche(address(srTranche)));
+
+        // since this flow doesn't go through pool.liquidateVault(),
+        // auctionsInProgress will be 0 when this liquidation is settled.
+        assertEq(pool.auctionsInProgress(), type(uint96).max);
```

Comment:
type(uint96).max?

Reply:
to check that auctionsInProgress is being updated. It's unchecked, so will underflow. It's a state that can't happen though